colbert/distillation/hf_scorers/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .bge_reranker import BGERerankerScorer
```

Comment:
do we need this here?

Reply:
Do you mean the init or bge models and hf scorers?

No in both. We can remove init file and we can remove hf scorers as whole too given they don't play any role as of now.